### ISSUE TITLE
Styling info type annotations gives invalid settings warning.

### ISSUE THREAD

**Issue by michaelgrohn** [SublimeLinter/SublimeLinter-annotations#31]
Annotations uses the `info` type for `#INFO`, `#NOTE`, etc.

Although these can be styled independly of `error` and `warning` like so…

```
{
   "linters": {
      "annotations": {
         "styles": [
            {
               "scope": "region.bluish markup.warning.sublime_linter",
               "types":[ "info" ]
            }
         ]
      }
   }
}

```

…this will make SublimeLinter show a warning at every startup:

```
Invalid settings in 'Packages/User/SublimeLinter.sublime-settings':
'linters' > 'annotations' > 'styles' > 'types': 'info' is not one of ['error', 'warning']
```

The style does apply correctly though, the startup waring is a false alarm. Seems like a limitation of SublimeLinter, that only `error` and `warning` are supposed to be used, but then why would the annotations plugin use types that are not "officially" supported?

**Comment by kaste** [comment]
No, that's a bug in SublimeLinter, we allow any `types` but never changed the settings validator.
